src/app/root.module.ts

```diff
@@ -12,9 +12,11 @@ import { NavigationModule } from './shared/navigation/navigation.module';
 
 export type CustomWindow = Window &
```

Comment:
@jaywalker21 Updating type of window here as well - https://github.com/razorpay/hypertrace-ui/blob/rzp_main/projects/common/src/telemetry/providers/freshpaint/load-snippet/index.d.ts#L14

Reply:
Noted, will push the change